src/main/java/com/dungeoncrawler/Javiarenka/characterCreator/CharacterCreatorService.java

```diff
@@ -0,0 +1,112 @@
+                        (StartingArmor.CLOTH_ARMOR.getName()),
+                        (StartingArmor.RUSTED_CHAIN_ARMOR.getName())
+                ));
+        characterClassToAvailableArmor.put(
+                HeroClass.WIZARD.toString(), Set.of(
+                        (StartingArmor.CLOTH_ARMOR.getName())
+                ));
+        characterClassToAvailableArmor.put(
+                HeroClass.ROGUE.toString(), Set.of(
+                        (StartingArmor.CLOTH_ARMOR.getName()),
+                        (StartingArmor.LEATHER_ARMOR.getName())
+                ));
+        characterClassToAvailableArmor.put(
+                HeroClass.HEALER.toString(), Set.of(
+                        (StartingArmor.CLOTH_ARMOR.getName())
+                ));
+        characterClassToAvailableArmor.put(
+                HeroClass.WARRIOR.toString(), Set.of(
+                        (StartingArmor.RUSTED_CHAIN_ARMOR.getName()),
+                        (StartingArmor.LEATHER_ARMOR.getName())
+                ));
+        characterClassToAvailableArmor.put(
+                HeroClass.KNIGHT.toString(), Set.of(
+                        (StartingArmor.RUSTED_CHAIN_ARMOR.getName())
+                ));
+        characterClassToAvailableWeapon.put(
+                HeroClass.WIZARD.toString(), Set.of(
+                        (StartingWeapon.STAFF_OF_FIRE.getName())
+                ));
+        characterClassToAvailableWeapon.put(
+                HeroClass.ARCHER.toString(), Set.of(
+                        (StartingWeapon.DAGGER.getName()),
+                        (StartingWeapon.SHORT_BOW.getName())
+                ));
+        characterClassToAvailableWeapon.put(
+                HeroClass.ROGUE.toString(), Set.of(
+                        (StartingWeapon.DAGGER.getName()),
+                        (StartingWeapon.SHORT_BOW.getName())
+                ));
+        characterClassToAvailableWeapon.put(
+                HeroClass.WARRIOR.toString(), Set.of(
+                        (StartingWeapon.DAGGER.getName()),
+                        (StartingWeapon.CLUB.getName()),
+                        (StartingWeapon.SHORT_BOW.getName()),
+                        (StartingWeapon.SHORT_SWORD.getName())
+                ));
+        characterClassToAvailableWeapon.put(
+                HeroClass.KNIGHT.toString(), Set.of(
+                        (StartingWeapon.CLUB.getName()),
+                        (StartingWeapon.SHORT_SWORD.getName())
+                ));
+        characterClassToAvailableWeapon.put(
+                HeroClass.HEALER.toString(), Set.of(
+                        (StartingWeapon.CLUB.getName())
+                ));
+        for (HeroClass hc : availableClasses) {
+            String classString = hc.toString();
+            classString = classString.substring(0, 1).toUpperCase() + classString.substring(1).toLowerCase();
+            availableClassesStringified.add(classString);
+        }
```

Comment:
To ma na celu "przełożenie" wszystkich klas na jednego Stringa i dodatkowo żeby każda nazwa zaczynała się z dużej litery?

Reply:
Prawie. Ten fragment nie przekłada wszystkich klas na jednego stringa, tylko robi z nich listę stringów z pierwszą wielką literą, a następnymi małymi, ,żeby potem w taki sposób wypisywać je w menu. Teraz niezależnie od tego jak będzie zbudowany w przyszłości kolejny enum, na froncie zawsze wyświetli nam się on ładnie